src/components/ContactInfo/ContactInfo.vue

```diff
@@ -293,7 +294,8 @@ export default class ContactInfo extends Vue {
     ]
     this.confirmEmailRules = [
       (v: string) => !!v || 'Confirm email address',
-      (v: string) => v.toString() === this.contactInfo.email || 'Email addresses must match'
+      (v: string) => !v || (v.toString() === (this.$refs.editContactForm && this.$refs.editContactForm.$el[0].value)) ||
```

Comment:
I worked at trying to get our storybook to recognize optional chaining but was unsuccessful, so i opted to short circuit here. 

Will continue to do a little research with regards to storybooks issue with the TS configuration (i think that's what it is at least)

Reply:
I, too, found that optional chaining didn't work in the bscs-shared-components project.

Maybe there's something missing from tsconfig.json ?

Good luck (may it work for all of us)!

Reply:
haha thanks! 

Yes, i believe you are right. Will take a little bit of research but will be nice to figure it out!